src/main/java/com/github/princesslana/slothbot/MessageCounter.java

```diff
@@ -81,6 +106,53 @@ public void clearBuckets(String channelId) {
                 .collect(ImmutableList.toImmutableList()));
   }
 
+  public boolean getBotCounterConfig(String guildId) {
+    return Optional.ofNullable(botCounterConfigs.get(guildId)).orElse(false);
+  }
+
+  public void setBotCounterConfig(String guildId, boolean value) {
+    if (value) {
+      botCounterConfigs.put(guildId, true);
+    } else {
+      botCounterConfigs.remove(guildId);
```

Comment:
why remove rather than set to false?

Reply:
Made sense to me initially since the default was false, but thinking about it now, I guess it's just unnecessary micro-ptimization, I'll set it to false instead.